packages/steveo/src/producers/sqs.ts

```diff
@@ -84,6 +85,22 @@ class SqsProducer extends BaseProducer implements IProducer {
       },
     };
 
+    // check if queue supports DLQ on the task config
+    let redrivePolicy: QueueAttributeMap | undefined;
+    if (task?.options?.deadLetterQueue) {
+      // Fetch existing or create new queue for DLQ
+      redrivePolicy = await this.getDeadLetterQueuePocily(
```

Comment:
mispelt policy

Reply:
can we just make this.getDeadLetterQueuePocily do the logic here?

Reply:
so have an early exit in the getDeadLetterQueuePocily method if deadLetterQuery is false/null